modules/core/site.js

```diff
@@ -2012,6 +2025,12 @@ $(function() {
         try { navigator.registerProtocolHandler("mailto", "?page=compose&compose_to=%s", "Cypht"); } catch(e) {}
     }
 
+    if (hm_page_name() == 'home') {
+        $('.pw_update').on("click", function() { update_password($(this).data('id')); });
+    }
+    if (hm_page_name() == 'servers') {
+        $('.edit_server_connection').on('click', imap_smtp_edit_action);
```

Comment:
You should do the same here. Check if the statement is not yet in the serversPageHandlers and move it to there.